src/nifgen/metadata/config.py

```diff
@@ -16,7 +16,7 @@
     'library_info': {
         'Linux': {
             '64bit': {
-                'name': 'libfgen.so',
+                'name': 'fgen',
```

Comment:
Please make this `nifgen`. Even though that driver has no Linux support, I am fairly sure the library would not be "fgen".